src/style.css

```diff
@@ -0,0 +1,57 @@
+.attribution-link {
+  position: relative;
+  top:0;
+  left: 0;
+  padding: 5px;
+  font-family: 'Raleway', sans-serif;
+}
+.wrapper {
+  width: 980px;
+  margin: 0 auto;
+}
+.main-title {
+  font-family: 'Playfair Display', serif;
+  font-size: 80px;
+  margin: 20px 10px;
+  color: #000;
+}
+#container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+.world-news {
```

Comment:
Same as JS variable naming. Better not to relate to data itself.
Use more general values like `items-list`, `item`, `card`, `panel`, etc.

Reply:
Renamed.